bower.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "eventdispatcher.js",
+  "version": "1.0.0",
+  "authors": [
+    "Rémi Becheras <rbecheras@sirap.fr>"
```

Comment:
Oh yes I'm sorry I worked on a fork in a private gitlab server and I generated the bower package from it so my name an my private repo was put in. I'll fix that soon